test/string-class-test.js

```diff
@@ -0,0 +1,175 @@
+var expect = require('chai').expect;
+require('../public/src/string-class');
+
+describe('String Class', () => {
+
+  describe('hasVowels', () => {
+    it('should return true for strings that has vowels in them', () => {
+      expect('word'.hasVowels()).to.be.true;
+      expect('andela'.hasVowels()).to.be.true;
+    });
+    it('should return true for strings that has vowels no matter the case of the word', () => {
+      expect('AndEla'.hasVowels()).to.be.true;
+    });
+    it('should return false for strings with consonants only', () => {
+      expect('cyst'.hasVowels()).to.be.false;
+      expect('Hymn'.hasVowels()).to.be.false;
+    });
+  });
+
+  describe('toUpper', () => {
+    it('should convert strings in lowercase to uppercase', () => {
+      expect('cyst'.toUpper()).to.equal('CYST');
+      expect('andela'.toUpper()).to.equal('ANDELA');
```

Comment:
### Nit-Pick
Repeating the same test twice, you can remove one of the two (line 22 and line 23)
Also in line 26 and 27